packages/evershop/src/lib/components/Area.js

```diff
@@ -61,7 +61,7 @@ Area.propTypes = {
     id: PropTypes.string,
     sortOrder: PropTypes.number,
     component: PropTypes.shape({
-      default: PropTypes.node
+      default: PropTypes.oneOfType([PropTypes.node, PropTypes.func])
```

Comment:
Hi @cagils ,

This is not really a function, it should be a React component. Is there any type can represent a React Component?

Reply:
I think you are right. I haven't found it before but there is the [PropTypes.elementType](https://reactjs.org/docs/typechecking-with-proptypes.html) exactly for this situation (either an element or component). I changed the type to this and the error is still resolved. Can you check?